index.html

```diff
@@ -2517,26 +2513,67 @@ <h3>
         <dl>
           <dt>@context</dt>
           <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. All members of the
+The value of the <code>@context</code> property MUST be exactly one of the following:
+
+<ul>
+  <li>The string <code>https://www.w3.org/ns/did/v1</code>. 
+    <pre class="example">
+      {
+        "@context": "https://www.w3.org/ns/did/v1",
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array of one or more elements,
```

Comment:
Another idea, how about something along the lines of following, need to make sure we refer to the JSON types correctly though

```
Further to this how about the following

```
An array of one or more elements, where the value types of the array members MUST either be a string or an object and the first element of the array is the string <code>https://www.w3.org/ns/did/v1</code>
```
```